ng2-components/ng2-alfresco-core/package.json

```diff
@@ -60,6 +60,7 @@
     "@angular/core": "2.2.2",
     "@angular/forms": "2.2.2",
     "@angular/http": "2.2.2",
+    "@angular/material": "^2.0.0-beta.1",
```

Comment:
I suggest pinning to the specific version. Who knows what they change in the upcoming beta.